Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -887,6 +879,12 @@ private void EnableAllServices()
 
         private void UpdateAllServices()
         {
+            // If the Mixed Reality Toolkit is not configured, stop.
```

Comment:
Curious, are these just fixing existing bugs, or is that needed specifically to get profile switching working?

Reply:
this change is intended to add the same protections to Update that exist in LateUpdate